### PR TITLE
[ZARAZ]Added custom fields

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses https://jira.cfops.it/browse/PCX-3155

### CODE REVIEW

File: products/zaraz/src/content/get-started/custom-fields.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Almost all tools in the Tools Library require that you enter information in more than one field to be properly configured. Beyond these required default fields, some tools let you add additional custom fields.
+
+If we take Universal Google Analytics as an example, a default field would be your GA Property ID (the Google Analytics account ID), and an example of a custom field would be a Custom Dimension (like Custom Dimension 1).
+
+If you add a custom field in the tool's configuration page, this field will be attached to every event that belongs to the tool. Alternatively, you can add custom fields to specific events that will only affect that event.
+
+## Add a custom field at the tool level (affects every event)
+
+1. Go to [Zaraz's main dashboard](https://dash.cloudflare.com/?to=/:account/:zone/zaraz).
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/custom-fields.md]: Any reason for not using our usual two first steps to log in, select account and website, and go to Zaraz dashboard?

File: products/zaraz/src/content/get-started/custom-fields.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+# Add custom fields to third-party tools
+
+Almost all tools in the Tools Library require that you enter information in more than one field to be properly configured. Beyond these required default fields, some tools let you add additional custom fields.
+
+If we take Universal Google Analytics as an example, a default field would be your GA Property ID (the Google Analytics account ID), and an example of a custom field would be a Custom Dimension (like Custom Dimension 1).
+
+If you add a custom field in the tool's configuration page, this field will be attached to every event that belongs to the tool. Alternatively, you can add custom fields to specific events that will only affect that event.
+
+## Add a custom field at the tool level (affects every event)
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/custom-fields.md]:
```suggestion
## Add a custom field at the tool level
```
It's already mentioned in the previous paragraph.

File: products/zaraz/src/content/get-started/custom-fields.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+1. Go to [Zaraz's main dashboard](https://dash.cloudflare.com/?to=/:account/:zone/zaraz).
+1. Locate the third-party tool where you want to add the custom field and click **Edit**.
+1. Click **Settings** > **Add Field**.
+1. Choose the desired field from the dropdown menu, and click **Add**.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/custom-fields.md]:
```suggestion
1. Choose the desired field from the drop-down menu, and click **Add**.
```

File: products/zaraz/src/content/get-started/custom-fields.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+The new field will now be attached to every event that belongs to the tool.
+
+## Add a custom field to a specific event
+
+1. Go to [Zaraz's main dashboard](https://dash.cloudflare.com/?to=/:account/:zone/zaraz).
+1. Locate the third-party tool where you want to add the custom field and click **Edit**.
+1. Click the event you wish to modify.
+1. Click **Add Field**.
+1. Choose the desired field from the dropdown menu and click **Add**.
```
Comment (pedrosousa) [inline products/zaraz/src/content/get-started/custom-fields.md]:
```suggestion
1. Choose the desired field from the drop-down menu and click **Add**.
```

**Review by pedrosousa — APPROVED** [review]

File: products/zaraz/src/content/get-started/custom-fields.md (hunk 1 above)
Comment (marciocloudflare) [reply products/zaraz/src/content/get-started/custom-fields.md]: No. I can't remember why I did it this way, tbh

**Pull request merged**